regional-go-service/README.md

```diff
@@ -0,0 +1,41 @@
+# `regional-go-service`
+
+DO NOT SUBMIT
```

Comment:
```suggestion
```

🤣

Reply:
This was so I wouldn't forget to come back and fill it out 😁